contrib/rs_into_django.py

```
@@ -0,0 +1,174 @@
+                    # Já existe, apenas atualiza a observação se necessário
+                    try:
+                        existing = Schedule.objects.get(
+                            parish=parish, day=day, start_time=start_time
+                        )
+                        if existing.observation != schedule_text:
+                            existing.observation = schedule_text
+                            existing.verified_at = datetime.now().date()
+                            existing.save()
+                            print(f"Horário atualizado: {existing}")
+                    except:
+                        pass
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)